### ISSUE TITLE
Change default pixel size to 2

### ISSUE THREAD

**Issue by falkamelung** [geodesymiami/insarmaps#113]
Currently the default size is at 10. It looks a bit weired at this size. Always, the first thing I am doing is reducing the size. Please change to 3.

**Comment by stackTom** [comment]
So always have it set to 3, even if it's a high resolution data set?

**Comment by falkamelung** [comment]
Yes, that is fine.

We could do differently but it may be complicated:

Ideally at all zoom levels the default pixel size is such that for adjacent pixels there is no free space in between (I mean no free space in the center of 4 adjacent pixels). If you know how to calculate this default pixel size, that would be preferred.  I think this would mean different default pixel sizes at different zoom levels. 

If a certain zoom level is reached we would set to a fixed value.

**Comment by stackTom** [comment]
Yes that is complicated. The current size/zoom values are a good compromise so there are no empty gaps at each zoom level. I am not sure there is a way to determine how many points are being rendered per distance from an mbtiles. At least when I first looked into this there wasn't a way. But that was at the very beginning of insarmaps so things might have changed. I'll just do size 3 for now. We can add a new feature request for the future if we deem it very important.

**Comment by stackTom** [comment]
Done, please try it on exos.
